apps/web/src/server/applications/components/form/form-applicant-components.controller.js

```diff
@@ -0,0 +1,342 @@
+import { findAddressListByPostcode } from '@planning-inspectorate/address-lookup';
```

Comment:
the files inside the folder /components/form are green because they're recognized as new by github.
however the content of these files is basically just the copy/paste of what was previously on the controllers. small changes was made but unfortunately it's impossible to spot them